src/jit/gentree.cpp

```diff
@@ -8446,11 +8446,17 @@ GenTree** GenTreeUseEdgeIterator::GetNextUseEdge() const
             }
 
         // Call, phi, and SIMD nodes are handled by MoveNext{Call,Phi,SIMD}UseEdge, repsectively.
+        //
+        // If FEATURE_MULTIREG_ARGS is enabled, so PUTARG_STK nodes also have special handling.
         case GT_CALL:
         case GT_PHI:
 #ifdef FEATURE_SIMD
         case GT_SIMD:
 #endif
+#if FEATURE_MULTIREG_ARGS
+        case GT_PUTARG_STK:
+#endif
+
```

Comment:
We do want to hit the `unreached()` in these cases, as that is exactly what we expect. I could move these closer to the default case or add another `unreached()`, but the latter seems unnecessary.